<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: &quot;Source Code Pro&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_sifrovani">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Šifrování</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zadej_otevreny">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Zadej otevřený text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filtrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sifrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Šifrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_desifrovani">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Dešifrování</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zadej_sifrovany">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Zadej šifrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_desifrovany">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Dešifrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_otevreny">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Otevřený text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_klic_a">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Klíč a:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_klic_b">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Klíč b:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_otevreny">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_a">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_b">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_sifrovany">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_filtrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_sifrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_desifrovany">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_otevreny">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>400</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_over">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ověření</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_zasifruj">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Zašifruj text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_desifruj">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Dešifruj text</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="overeni_ano">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Source Code Pro&quot;;
color: rgb(85, 255, 0);
text-align: center;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Source Code Pro'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="overeni_ne">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Source Code Pro&quot;;
color: rgb(255, 0, 0);
text-align: center;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Source Code Pro'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>540</y>
     <width>941</width>
     <height>101</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(255, 255, 255);</string>
   </property>
   <row>
    <property name="text">
     <string>Zdrojová abeceda</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Šifrovaná abeceda</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>0</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>8</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>9</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>10</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>11</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>12</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>13</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>14</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>16</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>17</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>18</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>19</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>20</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>21</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>22</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>23</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>24</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>25</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button_abeceda">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Abeceda</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
